src/config/backend/EdFi.DmsConfigurationService.Backend.Postgresql/Deploy/Scripts/0010_Insert_ResourceClaim.sql

```diff
@@ -0,0 +1,426 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+ALTER TABLE dmscs.ResourceClaim ALTER COLUMN Id DROP IDENTITY;
+
+INSERT INTO dmscs.ResourceClaim (Id, ResourceName, ClaimName)
+VALUES
+    (1,'types','http://ed-fi.org/ods/identity/claims/domains/edFiTypes'),
+    (2,'systemDescriptors','http://ed-fi.org/ods/identity/claims/domains/systemDescriptors'),
+    (3,'managedDescriptors','http://ed-fi.org/ods/identity/claims/domains/managedDescriptors'),
+    (4,'educationOrganizations','http://ed-fi.org/ods/identity/claims/domains/educationOrganizations'),
+    (5,'people','http://ed-fi.org/ods/identity/claims/domains/people'),
+    (6,'relationshipBasedData','http://ed-fi.org/ods/identity/claims/domains/relationshipBasedData'),
+    (7,'assessmentMetadata','http://ed-fi.org/ods/identity/claims/domains/assessmentMetadata'),
+    (8,'identity','http://ed-fi.org/ods/identity/claims/services/identity'),
+    (9,'educationStandards','http://ed-fi.org/ods/identity/claims/domains/educationStandards'),
+    (10,'primaryRelationships','http://ed-fi.org/ods/identity/claims/domains/primaryRelationships'),
+    (11,'surveyDomain','http://ed-fi.org/ods/identity/claims/domains/surveyDomain'),
+    (12,'schoolYearType','http://ed-fi.org/ods/identity/claims/schoolYearType'),
+    (13,'absenceEventCategoryDescriptor','http://ed-fi.org/ods/identity/claims/absenceEventCategoryDescriptor'),
+    (14,'academicHonorCategoryDescriptor','http://ed-fi.org/ods/identity/claims/academicHonorCategoryDescriptor'),
```

Comment:
I've notice this, depending on your source, but the URIs here don't include the schema segment, which they should for non-services, non-domain claims:

* http://ed-fi.org/ods/identity/claims/academicHonorCategoryDescriptor

... should be...

* http://ed-fi.org/ods/identity/claims/ed-fi/academicHonorCategoryDescriptor

If memory serves, the Integration Test Harness in the Ed-Fi ODS API project has logic to normalize these claim URIs. @vimayya?